app/src/FreeDeck.cpp

```diff
@@ -328,3 +335,27 @@ void postSetup() {
 	setGlobalContrast(configFile.read());
 	loadPage(0);
 }
+
+void checkTimeOut() {
+  unsigned long currentTime = millis();
+  if (currentTime - timeOutStartTime  >= TIMEOUT_TIME)
+  {
+    if(timeOut == false) switchScreensOff();
+  }
+}
+
+void switchScreensOff() {
+  timeOut = true;
+  for (uint8_t buttonIndex = 0; buttonIndex < BD_COUNT; buttonIndex++) {
+    setMuxAddress(buttonIndex, TYPE_DISPLAY);
+    delay(1);
+    oledFill(0);
+  }
+}
+
+void switchScreensOn() {
+  unsigned long currentTime = millis();
+  timeOut = false;
+  timeOutStartTime  = currentTime;
```

Comment:
you can directly write
`timeOutStartTime = millis();`
no need to store it in currentTime in between ;)